packages/react-sdk/src/components/Whiteboard/Draft/DraftShapeChild.tsx

```diff
@@ -46,6 +52,53 @@ export const DraftShapeChild = ({
   const { activeColor } = useLayoutState();
   const slideInstance = useWhiteboardSlideInstance();
   const { setActiveTool } = useLayoutState();
+  const shapeSizes = useAppSelector((state) => selectShapeSizes(state));
+  const dispatch = useAppDispatch();
+
+  const updateShapeSize = useCallback(() => {
+    if (startCoords === undefined || endCoords === undefined) {
+      // Nothing to do if there are not start or end coordinates
+      return;
+    }
+
+    const shapeSize = {
+      width: Math.abs(startCoords.x - endCoords.x),
+      height: Math.abs(startCoords.y - endCoords.y),
+    };
+    dispatch(setShapeSize({ kind, size: shapeSize }));
+  }, [dispatch, endCoords, kind, startCoords]);
+
+  const handleClick = useCallback(
+    (point: Point) => {
+      const { startCoords, endCoords } = calculateShapeCoords(
+        kind,
+        point,
+        shapeSizes,
+      );
+
+      slideInstance.addElement(
+        createShape({
+          kind,
+          startCoords,
+          endCoords,
+          fillColor: fixedColor || activeColor,
```

Comment:
With nullable values, I will always wonder if this shouldn't be `??` instead of `||`.

But if an empty string for `fixedColor` is the same as it not being defined, then `||` is correct.

Reply:
No annotation and no test file mention `fixedColor`. I'm actually not sure what it does.

Reply:
Found `fixedColor` here https://github.com/nordeck/matrix-neoboard/blob/c418ef6c89e3890b98cd8be2f552bad2470929a9/packages/react-sdk/src/components/Whiteboard/Draft/DraftPicker.tsx#L33

Updated the doc block.